factom_core/block_elements/chain_commit.py

```diff
@@ -13,25 +13,37 @@ class ChainCommit:
     entry_hash: bytes
     ec_spent: int
     ec_public_key: bytes
-    signature: bytes
+    signature: bytes = None
 
     def __post_init__(self):
         # TODO: value assertions
         pass
 
+    def set_signature(self, signature: bytes):
+        assert isinstance(signature, bytes)
+        self.signature = signature
+
```

Comment:
Yeah I meant just no checking at all for the signature setting. So we don't need this function in either of the files. How we handle type/value assertions will be determined when adding it everywhere in the same PR

Reply:
I misunderstood, sorry.